MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -46,6 +46,16 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
     _boundary_elements.push_back(new MeshLib::Point{nodes, node_ids[0]});
     for (auto const* bulk_element : _mesh.getNode(node_ids[0])->getElements())
     {
+        // Special treatment of the non-line elements because the identifyFace
+        // implementation expects three valid node pointers in the call. We can
+        // guarantee only one valid node, and calling the identifyFace would
+        // most likely lead to a SEGV.
```

Comment:
Maybe I don't understand it right, but isn't that necessary only for points then?

Reply:
It is for points only (see function name: boundaryElementsAtPoint), but maybe I missed you point ;)

Note to myself: a similar case happens when searching for lines (having two points) in an element's face... but the case didn't happen yet.

Reply:
if there is something more to discuss, I'll prepare another PR, but for now we have to move on...